symptoms/symptoms.js

```diff
@@ -0,0 +1,17 @@
+//dark mode 
+var icon = document.getElementById("icon");
+icon.onclick = function(){
+    document.body.classList.toggle("dark-mode");
+    if ( document.body.classList.contains("dark-mode")) {
+        icon.src="../assets/sun.svg";
+
+    }else{
+        icon.src="../assets/dark-mode.png";
+    }
+}
+document.getElementById("button").onclick = function () {
```

Comment:
Why didn't you assign your element to a const with meaningful name like icon?
if you may use your element in your code in future just assign it to const at the top of the code